From 2e48e4045e1e951433da0ca4b1e49798eedde14f Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 24 Apr 2018 12:21:18 -0700
Subject: [PATCH] Fix clang -Wreserved-user-defined-literal warnings

| us428control.cc:66:18: error: invalid suffix on literal; C++11 requires a space between literal and identifier [-Wreserved-user-defined-literal]
|         printf("usage: "PROGNAME" [-v verbosity_level 0..2] [-c card] [-D device] [-u usb-device] [-m mode]\n");
|                         ^

Upstream-Status: Submitted [https://patchwork.kernel.org/patch/10360805/]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 us428control/us428control.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/us428control/us428control.cc b/us428control/us428control.cc
index e839bf4..8cb3c42 100644
--- a/us428control/us428control.cc
+++ b/us428control/us428control.cc
@@ -63,7 +63,7 @@ static void usage(void)
 {
 	printf("Tascam US-428 Control\n");
 	printf("version %s\n", VERSION);
-	printf("usage: "PROGNAME" [-v verbosity_level 0..2] [-c card] [-D device] [-u usb-device] [-m mode]\n");
+	printf("usage: " PROGNAME " [-v verbosity_level 0..2] [-c card] [-D device] [-u usb-device] [-m mode]\n");
 	printf("mode is one of (us224, us428, mixxx)\n");
 }
 /*
